app/views/work_packages/edit.html.erb

```diff
@@ -29,7 +29,7 @@ See doc/COPYRIGHT.rdoc for more details.
 
 <% html_title l(:label_work_package_edit, name: work_package.to_s) %>
 
-<h2><%=h work_package.to_s %></h2>
+<h2 class="legacy-heading"><%=h work_package.to_s %></h2>
```

Comment:
TIL that I usually edit work packages on the show view of the work package.

As far as I can see the edit form (e.g. [this one](http://floriank-feature-18330-use-toolbar.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/work_packages/52/edit)) could easily use the new toolbar helper. Although I don't know if this form is linked anywhere.